test/components/Operations.test.js

```diff
@@ -59,7 +59,7 @@ describe('Operations component', () => {
     const expected = `
 ## Operations
 
-### SUB \`user/signedup\` Operation
+### PUB \`user/signedup\` Operation
```

Comment:
As you can see the template now displayed the action of the application instead of the opposite view.

Not sure if anything else has to change or whether we want to revert back to the old, not sure?

Reply:
Yup, we should keep it the same for v2. Only change it for v3+. Might be a good idea to introduce a flag to replace PUB/SUB with RECEIVE/SEND too. But the perspective should remain correct for both major versions.

Reply:
For a flag to switch between stuff, lets do that in a feature request if necessary, not adding more complexity here 🙂